core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3989,6 +3997,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.master.embedded.journal.addresses";
     public static final String MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT =
         "alluxio.master.embedded.journal.election.timeout";
+    public static final String MASTER_EMBEDDED_JOURNAL_SINGLE_MASTER_ELECTION_TIMEOUT =
+        "alluxio.master.embedded.journal.single.master.election.timeout";
```

Comment:
Does this value really need to be configurable? I think more apt behavior would be to respect the original timeout if a user configures it, but if not set in configuration (i.e. typical for local testing), just use a saner, smaller value

Reply:
Agreed.